tools/quake3/common/polylib.c

```diff
@@ -467,7 +469,7 @@ winding_t   *CopyWinding( winding_t *w ){
 	}
 
 	c = AllocWinding( w->numpoints );
-	size = (int)( (size_t)( (winding_t *)0 )->p[w->numpoints] );
+	size = offsetof( winding_t, p ) + sizeof( *w->p ) * w->numpoints;
```

Comment:
In a future pull request we can change `winding_t` to have a flexible array member, in which case we may replace the offset computation with a simple `sizeof`. But the present change is independent of that future change and less controversial, so I wanted to send it separately.